pkgs/top-level/aliases.nix

```
@@ -566,7 +566,8 @@ mapAliases ({
   telepathy_idle = telepathy-idle; # added 2018-02-25
   telepathy_logger = telepathy-logger; # added 2018-02-25
   telepathy_mission_control = telepathy-mission-control; # added 2018-02-25
-  telepathy_qt = telepathy-qt; # added 2018-02-25
+  telepathy-qt = libsForQt5.telepathy; # added 2020-07-01
+  telepathy_qt = libsForQt5.telepathy; # added 2018-02-25
```

I would just add

```suggestion
  telepathy-qt = throw "telepathy-qt no longer supports Qt4"; # added 2020-07-01
  telepathy_qt = telepathy-qt;
```

since it is a different package.

---

Makes sense. I wasn't too sure what to do with the Qt 4 version and it seemed like nothing within the packages depended on it, so I thought it was appropriate to alias to the Qt 5 version.